src/server/serverSetup.js

```diff
@@ -214,6 +226,26 @@ fluid.defaults("sjrk.storyTelling.server", {
     }
 });
 
+fluid.defaults("sjrk.storyTelling.server.https", {
+    httpsServerOptions: {
+        key: {
+            expander: {
+                func: "fs.readFileSync",
```

Comment:
If this fails, it will throw an exception which will probably bomb out the entire construction of the server

Reply:
Isn't that somewhat desirable as you wouldn't be able to run in HTTPS without it.

Reply:
OK, we should try this out to see whether the resulting diagnostic is intelligible

Reply:
It looks like the following:

```bash
08:03:22.957:  FATAL ERROR: Uncaught exception: ENOENT: no such file or directory, open './server-keys.pem'
Error: ENOENT: no such file or directory, open './server-keys.pem'
    at Object.openSync (fs.js:476:3)
    at readFileSync (fs.js:377:35)
    at fluid.expander.invokeFunc (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3498:21)
    at Object.fluid.expandExpander (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3405:16)
    at Object.fluid.expandSource (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3251:30)
    at Object.strategy (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3293:26)
    at /Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3213:29
    at Object.fluid.each (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/Fluid.js:655:17)
    at Object.fluid.fetchExpandChildren (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3206:15)
    at recurse (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3274:26)
    at Object.fluid.expandSource (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3261:13)
    at Object.strategy (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3293:26)
    at /Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3213:29
    at Object.fluid.each (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/Fluid.js:655:17)
    at Object.fluid.fetchExpandChildren (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3206:15)
    at Object.options.initter (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:3318:40)
    at fluid.concludeComponentObservation (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:1495:41)
    at /Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:2198:29
    at Array.forEach (<anonymous>)
    at localWorkflowTask (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:2197:41)
    at Object.invokeNext (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:258:56)
    at Function.fluid.promise.resumeSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:238:39)
    at Function.fluid.promise.progressSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:219:23)
    at Function.fluid.promise.resumeSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:247:31)
    at Function.fluid.promise.progressSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:219:23)
    at Function.fluid.promise.resumeSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:247:31)
    at Function.fluid.promise.progressSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:219:23)
    at Function.fluid.promise.resumeSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:247:31)
    at Function.fluid.promise.progressSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:219:23)
    at Function.fluid.promise.resumeSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:247:31)
    at Function.fluid.promise.progressSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:219:23)
    at Function.fluid.promise.resumeSequence (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidPromises.js:247:31)
    at commitPotentiae (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:2319:31)
    at Object.fluid.tryCatch (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/Fluid.js:243:24)
    at Object.fluid.commitPotentiae (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/FluidIoC.js:2304:15)
    at Object.fluid.initFreeComponent (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/Fluid.js:3024:28)
    at creator (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/Fluid.js:2306:26)
    at Object.fluid.invokeGlobalFunction (/Users/jo/Documents/git/sjrk-story-telling/node_modules/infusion/src/framework/core/js/Fluid.js:1355:25)
    at Function.kettle.config.loadConfig (/Users/jo/Documents/git/sjrk-story-telling/node_modules/kettle/lib/KettleConfigLoader.js:178:18)
    at Object.<anonymous> (/Users/jo/Documents/git/sjrk-story-telling/index.js:26:15)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
```